src/lib.rs

```diff
@@ -1186,14 +1163,14 @@ impl TestKitApi {
 
     /// Returns the mounting point for public APIs. Useful for intricate testing not covered
     /// by `get*` and `post*` functions.
-    pub fn public_mount(&self) -> &Mount {
-        &self.public_mount
+    pub fn public_handler(&self) -> &Chain {
```

Comment:
That wont' work, because there is now `impl<H: Handler> for Handler for &H`, so passing a trait object to, say, `iron_test::request::get` results in 

```
   --> tests/test_currency.rs:710:16
    |
710 |     let resp = request::get(&url, headers, api.public_handler()).unwrap();
    |                ^^^^^^^^^^^^ `iron::Handler` does not have a constant size known at compile-time
    |
    = help: the trait `std::marker::Sized` is not implemented for `iron::Handler`
    = note: required by `iron_test::request::get`
```

We probably will be able to use impl trait here though, once it is stable.

Reply:
Yeah, thought about it too after I've made the comment. The problem in the case you've mentioned is that `request::get` has signature `fn<H: Handler>(..., &H)`, which means `H` should be `Sized` (as a type param without `?Sized` opt-out). It seems to me that in this particular case `H` could be replaced with the `&Handler` or should be made `?Sized` (as it's taken by ref anyway), but I'm most probably missing something. Anyway, `&Chain` indeed works where `&Handler` doesn't, and that's all that matters.